lib/bridge/MatrixHandler.js

```diff
@@ -674,6 +678,20 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
             return;
         }
     }
+    else if (cmd === "!bridgeversion") {
+        try {
+            const BridgeVersion = require("../../package.json").version;
+            const msgText = `BridgeVersion: ${BridgeVersion}`;
```

Comment:
I think ``notice`` can be a const, or better yet do away with ``msgText`` and ``notice`` and make it part of sendMatrixAction parameter list.

Reply:
did you mean something like 
            yield this.ircBridge.sendMatrixAction(adminRoom, botUser, new MatrixAction("notice", msgText), req);
or			
			yield this.ircBridge.sendMatrixAction(adminRoom, botUser, new MatrixAction("notice", `BridgeVersion: ${BridgeVersion}`), req);

i can make a commit with it

Reply:
resolved with next Commit